.github/ISSUE_TEMPLATE/issue_review_template

```diff
@@ -0,0 +1,24 @@
+---
+name: 💡 Review 🌟
+about: Indicates that the review of a given issues or PR shall be done by the Release Mgmt team
+title: 'API release review request'
+labels: 'review'
+assignees: ''
+
+---
+
+**Related API issue or PR **
+<!-- Put here the link to the issue(s) or PR(s) that needs to be reviewed  -->
+
+**Expected actions**
+To review, follow the below list. 
+For any item, when done, put a comment in this issue.
+
+* yaml (check version in info & servers objects)  
+* test file(s) availability
+* changelog updated
+* readme updated (enforce correct release number / API version naming) 
+* API readiness checklist
```

Comment:
How about allowing to check-off the points also - GitHub will show then a nice progress indicator within the issue overview:

```suggestion
[ ] API Definition files (YAML) (check version in info & servers objects)  
[ ] test file(s) availability
[ ] changelog updated
[ ] readme updated (enforce correct release number / API version naming) 
[ ] API readiness checklist(s)
```